src/libraries/System.Private.CoreLib/src/System/IO/UnmanagedMemoryStream.cs

```diff
@@ -128,7 +127,6 @@ protected void Initialize(SafeBuffer buffer, long offset, long length, FileAcces
             unsafe
             {
                 byte* pointer = null;
-                RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
@akoeplinger, does mono still need these? If yes, anything under src/libraries/System.Private.Corelib should be reverted.

Reply:
The implementation for `RuntimeHelpers.PrepareConstrainedRegions()` is empty so I don't think it's needed.

Reply:
Ok, thanks, I wasn't sure if the VM recognized it specially as an intrinsic.